src/status_im/contexts/wallet/swap/swap_proposal/view.cljs

```diff
@@ -19,4 +19,8 @@
       "Swap confirmation"]
      [quo/button
       {:on-press #(rf/dispatch [:open-modal :screen/wallet.swap-set-spending-cap])}
-      "Set spending cap"]]))
+      "Set spending cap"]
```

Comment:
Strings should be wrapped in translations macro.

Reply:
This is a temporary screen, will be removed, so no need to translate strings here. Sorry to not clarify that!